varats/varats/plots/blame_diff_correlation_matrix.py

```diff
@@ -164,6 +212,12 @@ def plot(self, view_mode: bool) -> None:
             raise PlotDataEmpty
         df.sort_values(by=['time_id'], inplace=True)
 
+        if LOG.getLogger().isEnabledFor(LOG.INFO):
+            for x_var in variables:
+                for y_var in variables:
+                    if x_var != y_var:
+                        log_interesting_revisions(x_var, y_var, df.copy())
+
```

Comment:
```suggestion
        if LOG.isEnabledFor(logging.INFO):
            for x_var in variables:
                for y_var in variables:
                    if x_var != y_var:
                        log_interesting_revisions(x_var, y_var, df.copy())

```
My intention was not that this could runs out of the box, more the idea to put the code behind a logger check. I think these relates it to our logger.